### DISCUSSION TITLE
Alguém usando JAM Stack?

### DISCUSSION THREAD

**Discussion by luizbills** [frontendbr/forum#958] [Perguntas e Respostas]
Alguém anda desenvolvendo algo no modelo do [JAM Stack](https://jamstack.org/)? Pode falar um pouco dos desafios que você enfrentou ou que anda enfrentando?

**Comment by ribeiroevandro** [comment]
Interessante, não conhecia e vou dar uma olhada com mais calma sobre o assunto, parece interessante...

**Comment by julioc98** [comment]
Gostaria de saber também
